src/test/java/uk/gov/companieshouse/filinghistory/consumer/mapper/pretransform/AssociatedFilingNodeMapperTest.java

```diff
@@ -6,53 +6,58 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.api.delta.FilingHistory;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.TransformerTestingUtils;
 import uk.gov.companieshouse.filinghistory.consumer.transformrules.functions.FormatDate;
 
 @ExtendWith(MockitoExtension.class)
 class AssociatedFilingNodeMapperTest {
 
-    @InjectMocks
-    private AssociatedFilingNodeMapper associatedFilingNodeMapper;
+    private static final ObjectMapper objectMapper = TransformerTestingUtils.getMapper();
 
-    @Mock
-    private ObjectMapper objectMapper;
+    private AssociatedFilingNodeMapper associatedFilingNodeMapper;
     @Mock
     private FormatDate formatDate;
 
     @Mock
     private FilingHistory delta;
 
+    @BeforeEach
+    void setUp() {
+        associatedFilingNodeMapper = new AssociatedFilingNodeMapper(objectMapper, formatDate);
```

Comment:
Do we need this? The value can be assigned once in the declaration, assuming the mapper is stateless

Reply:
@cgregoryCH explained that this fixes a mock ordering issue